content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/_cq_design_dialog/.content.xml

```diff
@@ -74,7 +77,187 @@
                                         <datasource
                                             jcr:primaryType="nt:unstructured"
                                             sling:resourceType="core/wcm/components/embed/v1/datasources/embeddables"/>
+                                        <granite:data
+                                            jcr:primaryType="nt:unstructured"
+                                            cq-dialog-dropdown-showhide-target=".list-option-listfrom-showhide-target"/>
                                     </allowedEmbeddables>
+                                    <youtubeParameters
```

Comment:
I would rather we collect and add tabs for `embeddable`s dinamically (preferably conditioned by the `allowedEmbeddables` above) so that if somebody writes a new `embeddable` they don't have to add it to the main component design dialog.